Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1015,6 +1015,33 @@ extension BrowserViewController: BrowserToolbarDelegate {
         tabManager.selectedTab?.reload()
     }
 
+    func browserToolbarDidLongPressReload(browserToolbar: BrowserToolbarProtocol, button: UIButton) {
+        guard #available(iOS 9.0, *) else {
+            return
+        }
+
+        guard let tab = tabManager.selectedTab where tab.webView?.URL != nil else {
+            return
+        }
+
+        let toggleActionTitle: String
+        let desktopSite: Bool
+        if let userAgent = tab.lastCustomUserAgent where !userAgent.isEmpty {
+            toggleActionTitle = NSLocalizedString("Request Mobile Site", comment: "Action Sheet Button for Requesting the Mobile Site")
```

Comment:
Please remove this string and the string below from `UIConstants`. We have placeholder strings for these in that file so that they'd be exported and localized so that we can ship this feature sooner. Since you're using the strings here, the placeholders aren't needed anymore.